priv/www/api/index.html

```diff
@@ -403,6 +443,10 @@ <h2>Reference</h2>
         <td>X</td>
         <td></td>
         <td class="path">/api/exchanges/<i>vhost</i>/<i>name</i></td>
+        <td>management</td>
+        <td>User added to vhost<br/>
+          [PUT & DELETE] Configuration
```

Comment:
"Configuration" is not a role or permission in RabbitMQ. `configure` is a permission (for a single virtual host).